internal/ast/ast.go

```diff
@@ -4036,18 +4024,26 @@ type ImportDeclaration struct {
 	Attributes      *ImportAttributesNode // ImportAttributesNode. Optional
 }
 
-func (f *NodeFactory) NewImportDeclaration(modifiers *ModifierList, importClause *ImportClauseNode, moduleSpecifier *Expression, attributes *ImportAttributesNode) *Node {
+func (f *NodeFactory) newImportOrJSImportDeclaration(kind Kind, modifiers *ModifierList, importClause *ImportClauseNode, moduleSpecifier *Expression, attributes *ImportAttributesNode) *Node {
```

Comment:
minor nit: I think `newTypeAliasOrJSTypeAliasDeclaration` reads better than `newTypeOrJSTypeAliasDeclaration`.

Reply:
Thanks, that does read much better.